src/moduler/verktoylinje/navigasjonslinje.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PT from 'prop-types';
+import { connect } from 'react-redux';
+import { selectDialoger } from '../dialog/dialog-selector';
+import { dialogFilter } from '../filtrering/filter/filter-utils';
+import { LenkepanelBase } from 'nav-frontend-lenkepanel/lib';
+import { Panel } from 'nav-frontend-paneler';
+import { selectErVeileder } from '../identitet/identitet-selector';
+import DialogIkon from '../aktivitet/visning/underelement-for-aktivitet/dialog/DialogIkon';
+
+const DITTNAV_PATH = '/dittnav/';
+const DIALOG_PATH = '/arbeidsrettet-dialog';
+
+class Navigasjonslinje extends Component {
+    render() {
+        const { erVeileder, antallUlesteDialoger } = this.props;
+        if (erVeileder) {
+            return null;
+        } else {
+            return (
+                <Panel className="navigasjonslinje">
```

Comment:
Hvorfor trenger du panel her ? Gir den noe ekstra funksjonalitet som trengs for å få ønsket oppførsel?

Reply:
Fordi designsystemet sier at `Panel ` "brukes til å ramme inn innhold", noe som er det jeg ønsket. Jeg mente det i det minste ga bedre lesbarhet enn en kaskade av `<div>`.

Reply:
hvis det ikke gir noe ekstra funksjonalitet, så er det bedre med en div